torch/ao/quantization/_dbr/auto_trace.py

```diff
@@ -114,36 +117,51 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
             kwargs = kwargs if kwargs else {}
             hook_type = get_torch_function_hook_type(parent_module, func)
 
-            if first_call and hook_type is not HookType.OP_HOOKS:
-                qstate = getattr(parent_module, '_auto_quant_state', None)
-                if qstate:
-                    qstate.add_seen_op_type_without_op_hooks(func)
-
             if hook_type is HookType.OP_HOOKS:
-                qstate = parent_module._auto_quant_state  # type: ignore[attr-defined]
                 fqn = module_id_to_fqn[id(parent_module)] if parent_module else None
+                qstate = parent_module._auto_quant_state  # type: ignore[attr-defined]
```

Comment:
are we coupling tracing with quantization logic here?

Reply:
oh, sorry, could you explain more? not sure what you mean yet

Reply:
oh sorry, I mean we are querying auto_qaunt_state here in auto_trace.py, which means auto_trace is coupled with quantization and can't be applied to other transformations, e.g. numeric suite?

Reply:
oh, I see.  Yes, currently it is coupled.  There could be ways to reduce the coupling, but it's not something we have planned for this half.